packages/bsky/src/indexer/config.ts

```diff
@@ -86,6 +92,9 @@ export class IndexerConfig {
     assert(dbPostgresUrl)
     assert(redisHost || (redisSentinelName && redisSentinelHosts?.length))
     assert(indexerPartitionIds.length > 0)
+    assert(imgUriEndpoint)
```

Comment:
btw i think i'm gonna make these not required & if they're not setup, then just no abyss integration